test/cookbooks/os_prepare/recipes/default.rb

```diff
@@ -19,6 +19,7 @@
 include_recipe('os_prepare::iis_site')
 include_recipe('os_prepare::iptables') unless node['osprepare']['docker']
 include_recipe('os_prepare::x509')
+include_recipe('os_prepare::dh_params')
```

Comment:
Normally I'd add this line in alphabetical order, but the existing recipes are already out of order. Let me know if this requires adjustment.